helm/loki/templates/_helpers.tpl

```diff
@@ -286,3 +288,155 @@ http {
   }
 }
 {{- end }}
+
+{{/*
+/!\ Giantswarm override to work around a bug in azure where `use_federated_token` fails.
+/!\ This was copied from upstream and ˋuse_federated_token` was removed for azure`
```

Comment:
Having the link where this was copied from would be good also.

Reply:
ok, added.